### PR TITLE
Backend/docker

### PR DESCRIPTION
**Pull request by LynxHack**
# Changes
Brief summary of changes being made.

Add docker to spin up both redis and express containers at once

Run

```
docker-compose up --build
```

or if already built

```
docker-compose up
```

# Tickets
Tag the issues that this PR addresses.

### CODE REVIEW

File: backend/src/routes.ts [hunk 1]
```diff
@@ -10,4 +10,12 @@ router.get('/', (req, res) => {
   res.send('Hello world');
 })
 
+router.get('/findRooms/:roomid', (req, res) => {
```
Comment (srijonsaha) [inline backend/src/routes.ts]: We should make the routes RESTful. It should be /rooms/:roomid.

File: backend/src/routes.ts [hunk 2]
```diff
@@ -10,4 +10,12 @@ router.get('/', (req, res) => {
...
+  redisDB.getRoom(req.params.roomid);
+})
+
+router.post('/createRoom', (req, res) => {
```
Comment (srijonsaha) [inline backend/src/routes.ts]: It should be just /rooms

File: backend/src/db.ts [hunk 3]
```diff
@@ -6,7 +6,8 @@ export class DBS{
     client: redis.RedisClient;
     constructor(){
         this.port = 6379;
-        this.host = "127.0.0.1";
+        // this.host = "127.0.0.1";
```
Comment (srijonsaha) [inline backend/src/db.ts]: remove this

File: backend/src/db.ts [hunk 4]
```diff
@@ -29,4 +30,47 @@ export class DBS{
             console.log('GET result ->' + res);
         });
     }
+
+    // Create Room
+    createRoom(roomid:any, roominfo:any){
```
Comment (srijonsaha) [inline backend/src/db.ts]: we should change roomid/roominfo to appropriate types in the future instead of any

Comment (LynxHack) [reply backend/src/db.ts]: for roominfo, should the interface generally be in the same doc or split into another one as good practice for typescript?

Comment (LynxHack) [reply backend/src/db.ts]: also, still not entirely sure what we want in the room information object

Comment (srijonsaha) [reply backend/src/db.ts]: > for roominfo, should the interface generally be in the same doc or split into another one as good practice for typescript?

you can do either for now once we have to more stuff we we'll have to probably reorganize again

> also, still not entirely sure what we want in the room information object
let's just make it an interface with room name and room description for now

**Pull request merged**